hyperspy/_components/rc.py

```diff
@@ -16,24 +16,52 @@
 # You should have received a copy of the GNU General Public License
 # along with  HyperSpy.  If not, see <http://www.gnu.org/licenses/>.
 
-import numpy as np
+from hyperspy._components.expression import Expression
 
-from hyperspy.component import Component
+class RC(Expression):
 
+    r"""
+    RC function component (based on the time-domain capacitor voltage response 
+    of an RC-circuit)
+    
+    .. math::
 
-class RC(Component):
-
-    """
+        f(x) = V_\mathrm{0} + V_\mathrm{max} \left[1 - \mathrm{exp}\left(
+            -\frac{x}{\tau}\right)\right]
+    
+    ====================== ===========
```

Comment:
Please use the internal convention for the table for now and use #2148 to discuss these changes.

Reply:
done